comunidaddo/events/admin.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import admin
+from django import forms
+from .models import Event, Region
+
+class EventAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = Event
+        fields = '__all__'
+
+
+class EventAdmin(admin.ModelAdmin):
+    form = EventAdminForm
+    list_display = [
+        'slug', 'created', 'last_updated',
+        'title', 'description', 'url', 'image',
+        'start_date', 'end_date', 'publish',
+        'publish_date', 'address', 'country', 'city']
+    readonly_fields = [
+        'slug', 'created', 'last_updated',
```

Comment:
Because they are fields generated by the system and I think it would not be good if they are changed by the admin because if that were the case the field loses credibility.

Example: if I create an event at a time X and then that time is modified, that data is false because in fact it was not created at that time, the same happens with the last_update field.